model/Dataset/Dataset.md

```diff
@@ -15,3 +15,14 @@ The Dataset namespace defines tbd.
 - id: https://rdf.spdx.org/v3/Dataset
 - name: Dataset
 
+## Profile conformance 
+
+For an element collection to be conformant with this profile,
+the following has to hold:
+
+1. for every `/Dataset/DatasetPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `concludedLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
+2. for every `/Dataset/DatasetPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `declaredLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
```

Comment:
Are you sure this is what you want?

Even the Licensing profile does not require a `declaredLicense`. Keep in mind that `declaredLicense` has to be *inside* the artifact. So, for example, information on a website does *not* qualify as such. Do `DatasetPackage`s usually include license inside?

Reply:
Discussed in AI meeting, and decided we want to go forward with declaredLicense